### ISSUE TITLE
Can Neuroconductor extract *monkey* brain?

### ISSUE THREAD

**Issue by hegde00** [259mit/NeuroConductor#1]
Hello,

We have a T1 MR image of a Rhesus monkey (Macaca mulatta). We need to extract the brain. Is is possible to use (any tool in) Neuroconductor to do this?  If not, any suggestions as to what other tool we could use, please?   Thank you very much in advance!

**Comment by mitheshdocs** [comment]
Hello.

Neuroconductor has certain packages that can get the job done for you.
Fslr is one such package that helps with extracting the brain.
You can refer this link:
https://neuroconductor.org/tutorials/brain_extraction

I also suggest you check this course by JHU: NeuroHacking in R. Its available on Coursera. I have created a folder in the main branch with the course notes pdfs for your reference.

I'll be happy to help if you need any further assistance.

Best regards,
Mithesh

**Comment by mitheshdocs** [comment]
I'm keeping this issue open.

**Comment by hegde00** [comment]
Thank you very much for the info and for keeping this issue open. I've looked at the brain extraction package within NeuroConductor you mentioned.

But I'm afraid it does **not** address our problem at hand, because this package addresses *human* brain extraction. (We already know how to extract *human* brains; we do it all the time in my lab. Sorry I wasn't clear about this in my original post.)

What we need to do is to find a freeware tool to extract a *monkey* brain (specifically, the brain of a Rhesus macaque). In order to do this, we'd need (among other things) the *template* of a *monkey* brain. A commonly used template is the so-called M12 template, but there are others. We could, in principle, acquire an M12 template and plug into your brain extraction toolkit, but this is obviously would not a be previously validated procedure. (I'm sure the tool in question is validated for *human* brain extraction, but it's not clear to us if it's validated for *monkey* brain extraction.)

So with your indulgence, let me rephrase my question thusly: Is there a *validated* tool within NeuroConductor to extract *monkey* brain?

Thank you in advance for your help,
Jay
